Code/Editor/CryEditDoc.cpp

```diff
@@ -108,21 +108,11 @@ namespace Internal
 // CCryEditDoc construction/destruction
 
 CCryEditDoc::CCryEditDoc()
-    : doc_validate_surface_types(0)
-    , m_modifiedModuleFlags(eModifiedNothing)
-    // It assumes loaded levels have already been exported. Can be a big fat lie, though.
-    // The right way would require us to save to the level folder the export status of the
-    // level.
-    , m_boLevelExported(true)
-    , m_modified(false)
-    , m_envProbeHeight(200.0f)
-    , m_envProbeSliceRelativePath("EngineAssets/Slices/DefaultLevelSetup.slice")
+    : m_modifiedModuleFlags(eModifiedNothing)
```

Comment:
question: Could this also be moved to a member initializer instead of being in the initializer list as we moved all the others?

Reply:
That enum is defined in `Code/Editor/IEditor.h` but that header is not included in `CryEditDoc.h`, I'm very wary of adding additional `#include` to a header file.

Reply:
Ah I see got you, okay makes sense thanks for clarifying :+1: